public/locales/az/gamepage.json

```diff
@@ -94,13 +97,22 @@
             "start": "İndi oyna",
             "stop": "Oynanır (Dayandır)"
         },
+        "queue": {
+            "remove": "Remove from Queue"
+        },
         "saves": {
             "syncing": "Sinxronizasiya Saxlamalar"
         }
     },
     "launch": {
         "options": "Başlama Seçimləri..."
     },
+    "Open": "",
+    "queue": {
+        "label": {
+            "remove": "Remove from download manager"
```

Comment:
We have this `queue.label.remove` and also `label.queue.remove` with the value `Remove from Queue`, maybe we can unify them?